bok_choy/query.py

```diff
@@ -445,6 +445,24 @@ def invisible(self):
         """
         return self.present and not self.visible
 
+    def is_focused(self):
+        """
+        Checks that at least one matched element is focused. More
+        specifically, if checks whether the element is document.activeElement.
+
+        Returns:
+            bool
```

Comment:
This will return False if no element matches the browser query, right?
We should make that clear in the docstring, because I know that people had questions about that case when using the visible() method.

I think either with a code example or further explanation in the Returns section would be fine.